app/controllers/users_controller.rb

```diff
@@ -23,10 +28,63 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update user_params
+      flash[:success] = t ".update.success"
+      redirect_to @user
+    else
+      flash.now[:danger] = t ".update.failed"
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user&.destroy
```

Comment:
```suggestion
    if @user.destroy
```
ko cầ `&.` nữa em nhé, vì lúc này đảm bảo user đã khác nil rồi

Reply:
à dạ, e quên mất 😅